tests/integration/sync_test.rs

```diff
@@ -93,15 +90,15 @@ fn test_sync_up() {
     let includes: Option<Vec<String>> = Some(vec!["*.txt".to_string()]);
     let excludes: Option<Vec<String>> = Some(vec!["*".to_string()]);
 
-    let res = blocking::sync(
-        s3client.as_ref(),
-        SyncDirection::up,
-        crate::TEST_BUCKET,
-        &s3_key,
-        &local_directory,
-        &includes,
-        &excludes,
-    );
+    let source = SyncParam::Local {
```

Comment:
Let's add constructors for Local and Bucket to get rid of some of this boilerplate